src/compositor/lipstickcompositor.cpp

```diff
@@ -704,7 +707,9 @@ void LipstickCompositor::setUpdatesEnabled(bool enabled)
             if (QWindow::handle()) {
                 QGuiApplication::platformNativeInterface()->nativeResourceForIntegration("DisplayOff");
             }
+            m_fakeRepaintLoop.start();
```

Comment:
Is there any signal readily available in surface that could be used to trigger the fakerepaintloop, instead of steady 10Hz timer? No biggie, though.